test/python/algorithms/test_measure_error_mitigation.py

```diff
@@ -44,14 +47,17 @@ def test_measurement_error_mitigation_with_diff_qubit_order(self):
         read_err = noise.errors.readout_error.ReadoutError([[0.9, 0.1], [0.25, 0.75]])
         noise_model.add_all_qubit_readout_error(read_err)
 
+        fitter_cls = (
```

Comment:
Should we add a test setting the `mit_pattern` in the `QuantumInstance`?

Reply:
Test added setting mit_pattern